packages/@aws-cdk/aws-logs/lib/log-group.ts

```diff
@@ -363,6 +371,7 @@ export class LogGroup extends LogGroupBase {
     }
 
     const resource = new CfnLogGroup(this, 'Resource', {
+      kmsKeyId: props.encryptionKey?.keyId,
```

Comment:
should this be using the ARN? 
The [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html#cfn-logs-loggroup-kmskeyid) indicate that it can. It would be more robust to use an ARN because the key might be in another account (some services allow it, I'm not sure about whether that's valid usage in this case)

Reply:
I agree. Let me make that adjustment also.